fsleyes_plugin_shimming_toolbox/tabs/b0shim_tab.py

```diff
@@ -214,66 +213,38 @@ def create_sizer_dynamic_shim(self, metadata=None):
             },
         ]
 
-        dropdown_scanner_format1 = DropdownComponent(
+        dropdown_scanner_format = DropdownComponent(
             panel=self,
             dropdown_metadata=dropdown_scanner_format_metadata,
             option_name='output-file-format-scanner',
             label="Scanner Output Format",
             cli=dynamic_cli
         )
 
-        dropdown_scanner_format2 = DropdownComponent(
-            panel=self,
-            dropdown_metadata=dropdown_scanner_format_metadata,
-            option_name='output-file-format-scanner',
-            label="Scanner Output Format",
-            cli=dynamic_cli
-        )
-
-        dropdown_scanner_format3 = DropdownComponent(
-            panel=self,
-            dropdown_metadata=dropdown_scanner_format_metadata,
-            option_name='output-file-format-scanner',
-            label="Scanner Output Format",
-            cli=dynamic_cli
-        )
-
-        dropdown_scanner_order_metadata = [
+        checkbox_scanner_order_metadata = [
```

Comment:
Consider using the metadata (option_value) to fill in the get_command() function rather than hardcoding the get_command function for the checkboxes.